e2e/api/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM opensearchstaging/remote-vector-index-builder:faiss-base-latest
-
-WORKDIR /e2e
+USER root
```

Comment:
should we revert back to appuser in this file?

Reply:
Fixed this
Might need to fix the core and app images as well
I'll take a look at it

Reply:
Works
E2e image runs even if we switch to appuser
Did similar changes to core and app as well